lib/reports/brexit_subscribers_report.rb

```diff
@@ -4,16 +4,18 @@ class Reports::BrexitSubscribersReport
   CSV_HEADERS = %w[title slug tags subscribed unsubscribed immediately daily weekly].freeze
   attr_reader :date
 
-  def self.call(*args)
-    new(args).export_csv
+  def call
+    export_csv
   end
 
+private
+
   def initialize(date = nil)
     @date = date
   end
```

Comment:
This seems a little surprising and unconventional to be private - I'd suggest moving to be public. 

Also a today-I-learnt moment that this works as private 😮